### PR TITLE
CI: Use ancient GCC to build for 32bit

### PR DESCRIPTION
**Pull request by dnnr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -87,6 +87,8 @@ jobs:
                 --volume=$HOME/.ccache-gcc-4.8:/.ccache:rw
                 --env CC=gcc-4.8
                 --env CXX=g++-4.8
+                --env CXXFLAGS=-m32
```
Comment (blarz) [inline .travis.yml:90]: As Werror is disabled in debugflags.cmake and we explictly enable it in build.py (which is not used here), we should add it here too to catch actual build errors.

Comment (blarz) [reply .travis.yml:90]: Also we are not doing a Debug build here, apparently. Not sure if this is really desired?

Comment (t-wissmann) [reply .travis.yml:90]: Not sure about that. For me it's important that herbstluftwm builds for people with old systems. Hence, I wouldn't care so much about `-Werror` and debug build.

Comment (dnnr) [reply .travis.yml:90]: I tend to agree with @t-wissmann here. On the other hand, it probably doesn't hurt either?

Comment (dnnr) [reply .travis.yml:90]: In any case, it'd be separate PR. :-)

Comment (blarz) [reply .travis.yml:90]: Ok then!

**Pull request merged**